src/TestUtils/src/Microsoft.Maui.IntegrationTests/Utilities/DotnetInternal.cs

```diff
@@ -7,7 +7,7 @@ public static class DotnetInternal
 	{
 		static readonly string DotnetRoot = Path.Combine(TestEnvironment.GetMauiDirectory(), ".dotnet");
 		static readonly string DotnetTool = Path.Combine(DotnetRoot, "dotnet");
-		const int DEFAULT_TIMEOUT = 900;
+		const int DEFAULT_TIMEOUT = 1800;
```

Comment:
Why requires to duplicate the default timeout time?

Reply:
BEcause we are hitting the limit . 

@rolfbjarne  @dalexsoto  i m  not sure what follow up we need here, but seems that we a issue where this is slower on net9 , and takes mroe than 15 minutes to do a simple release run . We have see other people saying this is getting slower to do dotnet run on net9

Reply:
Can you get a binlog of something that runs fast in .NET 8 and slow in .NET 9 (so two binlogs - before and after)?

Reply:
Ok this is from the test failure timeout on net9 [sample.binlog.zip](https://github.com/user-attachments/files/17653962/sample.binlog.zip)

I m trying to get the same for net8, we didn't had logs for this on that branch

Reply:
This seems it didn t work . 30 minutes it still hangs